core/src/main/scala/org/apache/spark/Accumulable.scala

```diff
@@ -49,6 +48,7 @@ import org.apache.spark.util.Utils
  * @tparam R the full accumulated data (result type)
  * @tparam T partial data that can be added in
  */
+@deprecated("use AccumulatorV2", "2.0.0")
```

Comment:
OK, I'm making another PR to try to fix build warnings. Deprecating these methods actually makes many warnings go away (since deprecated methods calling deprecated methods is OK). Is that much OK to do or don't touch it for now? these methods are going to stay but be deprecated in any event right?